LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/parsing/scrollbar-gutter-invalid-expected.txt

```diff
@@ -13,4 +13,11 @@ PASS e.style['scrollbar-gutter'] = "3em" should not set the property value
 PASS e.style['scrollbar-gutter'] = "1 2 3" should not set the property value
 PASS e.style['scrollbar-gutter'] = "none" should not set the property value
 PASS e.style['scrollbar-gutter'] = "red" should not set the property value
+PASS e.style['scrollbar-gutter'] = "stable both" should not set the property value
+PASS e.style['scrollbar-gutter'] = "stable force" should not set the property value
+PASS e.style['scrollbar-gutter'] = "stable both force" should not set the property value
+PASS e.style['scrollbar-gutter'] = "always" should not set the property value
+PASS e.style['scrollbar-gutter'] = "always both" should not set the property value
+PASS e.style['scrollbar-gutter'] = "always force" should not set the property value
+PASS e.style['scrollbar-gutter'] = "always both force" should not set the property value
```

Comment:
It would be interesting to test some bad combinations with `both-edges`

Reply:
Should I do that as a PR in the WPT repo or here?

Reply:
Here, and then exported to WPT would be ideal

Reply:
Have added those.

When I try to run the export script I get this

`python3 Tools/Scripts/export-w3c-test-changes`

"No changes to upstream. Exiting..."

Reply:
`export-w3c-test-changes -g HEAD -b 256892 --no-linter -c`

Reply:
https://github.com/web-platform-tests/wpt/pull/40250